### ISSUE TITLE
Cannot find module 'debug/browser'

### ISSUE THREAD

**Issue by birgersp** [hax/refactor#37]
[Enter steps to reproduce below:]

1. Install package

**Atom Version**: 1.12.7
**Electron Version**: 1.3.13
**System**: Microsoft Windows 10 Pro
**Thrown From**: [refactor](https://github.com/hax/refactor) package, v0.11.2

### Stack Trace

Failed to load the refactor package

```
At Cannot find module 'debug/browser'

Error: Cannot find module 'debug/browser'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (file:///C:/Users/birger/.atom/packages/refactor/lib/watcher.coffee:2:6)
    at Object.<anonymous> (file:///C:/Users/birger/.atom/packages/refactor/lib/watcher.coffee:1:1)
    at Module._compile (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .coffee] (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (file:///C:/Users/birger/.atom/packages/refactor/lib/refactor.coffee:2:11)
    at Object.<anonymous> (file:///C:/Users/birger/.atom/packages/refactor/lib/refactor.coffee:1:1)
    at Module._compile (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .coffee] (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package.js:747:27)
    at C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package.js:118:28
    at Package.module.exports.Package.measure (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package.js:92:15)
    at Package.module.exports.Package.load (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package.js:106:12)
    at PackageManager.module.exports.PackageManager.loadPackage (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package-manager.js:468:14)
    at PackageManager.module.exports.PackageManager.activatePackage (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package-manager.js:548:30)
    at C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\node_modules\settings-view\lib\package-manager.js:439:29
    at exit (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\node_modules\settings-view\lib\package-manager.js:73:16)
    at triggerExitCallback (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\buffered-process.js:215:47)
    at ChildProcess.<anonymous> (C:\Users\birger\AppData\Local\atom\app-1.12.7\resources\app.asar\src\buffered-process.js:237:18)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
```

### Commands

```
     -0:10.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "wrap-guide"
    ],
    "telemetryConsent": "no"
  }
}
```

### Installed Packages

```coffee
# User
atom-darcula, v0.2.0 (inactive)
atom-darcula-syntax, v0.1.0 (inactive)
atom-format-lua, v0.1.5 (inactive)
autocomplete-bash-builtins, v0.3.4 (active)
autocomplete-lua, v0.8.0 (active)
language-batch, v0.3.3 (active)
language-lua, v0.9.11 (active)
language-matlab, v0.2.1 (active)
minimap, v4.25.7 (active)
refactor, v0.11.2 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.45.0 (inactive)
base16-tomorrow-dark-theme, v1.3.0 (inactive)
base16-tomorrow-light-theme, v1.3.0 (inactive)
one-dark-ui, v1.6.2 (active)
one-light-ui, v1.6.2 (inactive)
one-dark-syntax, v1.5.0 (active)
one-light-syntax, v1.5.0 (inactive)
solarized-dark-syntax, v1.0.5 (inactive)
solarized-light-syntax, v1.0.5 (inactive)
about, v1.7.0 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.13.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.4 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.42.0 (active)
bracket-matcher, v0.82.2 (active)
command-palette, v0.39.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.202.2 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.158.8 (active)
metrics, v1.1.2 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.1 (inactive)
settings-view, v0.243.1 (active)
snippets, v1.0.4 (active)
spell-check, v0.68.4 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.2 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.0 (active)
timecop, v0.33.2 (active)
tree-view, v0.210.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.35.0 (active)
wrap-guide, v0.38.2 (inactive)
language-c, v0.54.0 (active)
language-clojure, v0.22.1 (active)
language-coffee-script, v0.48.0 (active)
language-csharp, v0.13.0 (active)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.43.0 (active)
language-html, v0.47.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (active)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (active)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```

**Comment by gdoublev** [comment]
Encountering same issue.

**Comment by jgrauel** [comment]
I am as well

**Comment by atomicwrites** [comment]
Me too
<details>
  <summary>Bug Report</summary>
  <p>

**Atom Version**: 1.12.7
**Electron Version**: 1.3.13
**System**: Microsoft Windows 10 Home
**Thrown From**: [refactor](https://github.com/hax/refactor) package, v0.11.2


### Stack Trace

Failed to load the refactor package

```
At Cannot find module 'debug/browser'

Error: Cannot find module 'debug/browser'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (file:///C:/Users/vbsma/.atom/packages/refactor/lib/watcher.coffee:2:6)
    at Object.<anonymous> (file:///C:/Users/vbsma/.atom/packages/refactor/lib/watcher.coffee:1:1)
    at Module._compile (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .coffee] (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (file:///C:/Users/vbsma/.atom/packages/refactor/lib/refactor.coffee:2:11)
    at Object.<anonymous> (file:///C:/Users/vbsma/.atom/packages/refactor/lib/refactor.coffee:1:1)
    at Module._compile (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .coffee] (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package.js:747:27)
    at C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package.js:118:28
    at Package.module.exports.Package.measure (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package.js:92:15)
    at Package.module.exports.Package.load (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package.js:106:12)
    at PackageManager.module.exports.PackageManager.loadPackage (C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package-manager.js:468:14)
    at C:\Users\vbsma\AppData\Local\atom\app-1.12.7\resources\app.asar\src\package-manager.js:412:19
```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "keybinding-resolver",
      "language-c",
      "language-clojure",
      "language-csharp",
      "language-go",
      "language-java",
      "language-mustache",
      "language-objective-c",
      "language-perl",
      "language-php",
      "language-property-list",
      "language-ruby",
      "language-ruby-on-rails",
      "language-shellscript",
      "language-toml",
      "symbols-view",
      "color-picker",
      "atomic-chrome",
      "atom-ternjs",
      "linter",
      "atom-jinja2",
      "language-latex",
      "latextools",
      "pdf-view",
      "standardjs-snippets",
      "hex",
      "autocomplete-python"
    ],
    "telemetryConsent": "limited",
    "themes": [
      "isotope-ui",
      "genesis-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
an-old-hope-syntax, v0.5.0 (inactive)
atom-jinja2, v0.6.0 (inactive)
atom-material-syntax, v0.4.6 (inactive)
atom-ternjs, v0.16.1 (inactive)
autocomplete-html-entities, v0.1.0 (inactive)
autocomplete-python, v1.8.20 (inactive)
genesis-syntax, v1.0.7 (inactive)
hex, v0.6.2 (inactive)
isotope-light-ui, v2.2.6 (inactive)
isotope-ui, v2.7.1 (inactive)
language-batch, v0.3.3 (inactive)
language-latex, v1.0.0 (inactive)
latextools, v0.8.5 (inactive)
linter, v1.11.18 (inactive)
pdf-view, v0.54.0 (inactive)
pigments, v0.37.0 (inactive)
proton-framer, v0.11.0 (inactive)
proton-light, v0.2.0 (inactive)
rad-syntax, v0.4.0 (inactive)
refactor, v0.11.2 (inactive)
seti-syntax, v1.0.1 (inactive)
seti-ui, v1.4.0 (inactive)
standardjs-snippets, v2.5.0 (inactive)
sync-settings, v0.8.0 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.45.0 (inactive)
base16-tomorrow-dark-theme, v1.3.0 (inactive)
base16-tomorrow-light-theme, v1.3.0 (inactive)
one-dark-ui, v1.6.2 (inactive)
one-light-ui, v1.6.2 (inactive)
one-dark-syntax, v1.5.0 (inactive)
one-light-syntax, v1.5.0 (inactive)
solarized-dark-syntax, v1.0.5 (inactive)
solarized-light-syntax, v1.0.5 (inactive)
about, v1.7.0 (inactive)
archive-view, v0.62.0 (inactive)
autocomplete-atom-api, v0.10.0 (inactive)
autocomplete-css, v0.13.1 (inactive)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.31.4 (inactive)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.42.0 (inactive)
bracket-matcher, v0.82.2 (inactive)
command-palette, v0.39.0 (inactive)
deprecation-cop, v0.54.1 (inactive)
dev-live-reload, v0.47.0 (inactive)
encoding-selector, v0.22.0 (inactive)
exception-reporting, v0.40.0 (inactive)
find-and-replace, v0.202.2 (inactive)
fuzzy-finder, v1.4.0 (inactive)
git-diff, v1.1.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (inactive)
image-view, v0.60.0 (inactive)
incompatible-packages, v0.26.1 (inactive)
keybinding-resolver, v0.35.0 (inactive)
line-ending-selector, v0.5.0 (inactive)
link, v0.31.2 (inactive)
markdown-preview, v0.158.8 (inactive)
metrics, v1.1.2 (inactive)
notifications, v0.65.1 (inactive)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.1 (inactive)
settings-view, v0.243.1 (inactive)
snippets, v1.0.4 (inactive)
spell-check, v0.68.4 (inactive)
status-bar, v1.4.1 (inactive)
styleguide, v0.47.2 (inactive)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.0 (inactive)
timecop, v0.33.2 (inactive)
tree-view, v0.210.0 (inactive)
update-package-dependencies, v0.10.0 (inactive)
welcome, v0.35.1 (inactive)
whitespace, v0.35.0 (inactive)
wrap-guide, v0.38.2 (inactive)
language-c, v0.54.0 (inactive)
language-clojure, v0.22.1 (inactive)
language-coffee-script, v0.48.0 (inactive)
language-csharp, v0.13.0 (inactive)
language-css, v0.40.1 (inactive)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (inactive)
language-go, v0.43.0 (inactive)
language-html, v0.47.1 (inactive)
language-hyperlink, v0.16.1 (inactive)
language-java, v0.24.0 (inactive)
language-javascript, v0.122.0 (inactive)
language-json, v0.18.3 (inactive)
language-less, v0.29.6 (inactive)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.37.0 (inactive)
language-php, v0.37.3 (inactive)
language-property-list, v0.8.0 (inactive)
language-python, v0.45.1 (inactive)
language-ruby, v0.70.2 (inactive)
language-ruby-on-rails, v0.25.1 (inactive)
language-sass, v0.57.0 (inactive)
language-shellscript, v0.23.0 (inactive)
language-source, v0.9.0 (inactive)
language-sql, v0.25.0 (inactive)
language-text, v0.7.1 (inactive)
language-todo, v0.29.1 (inactive)
language-toml, v0.18.1 (inactive)
language-xml, v0.34.12 (inactive)
language-yaml, v0.27.1 (inactive)

# Dev
No dev packages
```
</p>
</details>

**Comment by bbrzoska** [comment]
Me too, happens immediately after installing / activating.

<details>

**Atom Version**: 1.12.7
**Electron Version**: 1.3.13
**System**: Mac OS X 10.11.1
**Thrown From**: [refactor](https://github.com/hax/refactor) package, v0.11.2


### Stack Trace

Failed to load the refactor package

```
At Cannot find module 'debug/browser'

Error: Cannot find module 'debug/browser'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/bazyli.brzoska/.atom/packages/refactor/lib/watcher.coffee:2:6)
    at Object.<anonymous> (/Users/bazyli.brzoska/.atom/packages/refactor/lib/watcher.coffee:1:1)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/bazyli.brzoska/.atom/packages/refactor/lib/refactor.coffee:2:11)
    at Object.<anonymous> (/Users/bazyli.brzoska/.atom/packages/refactor/lib/refactor.coffee:1:1)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:747:27)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:118:28
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at Package.module.exports.Package.load (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:106:12)
    at PackageManager.module.exports.PackageManager.loadPackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:468:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:548:30)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:439:29
    at exit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:73:16)
    at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:215:47)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:229:18
    at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:100:18)
    at emitOne (events.js:101:20)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:493:12)
```

### Commands

```
     -1:20.8.0 editor:newline (atom-text-editor.editor.is-focused)
  2x -1:20.6.0 autocomplete-plus:activate (atom-text-editor.editor.is-focused)
  2x -1:18.4.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:16.7.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -1:10.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -1:10.6.0 autocomplete-plus:activate (atom-text-editor.editor)
     -1:06.3.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -1:01.6.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:00.0 bookmarks:jump-to-next-bookmark (atom-text-editor.editor.is-focused)
     -0:57.1.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:54.7.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:54.7.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:53.8.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:51.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:51.2.0 settings-view:open (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".hgrc",
        ".flowconfig"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "js-refactor",
      "coffee-refactor",
      "autocomplete-flow",
      "flow-ide",
      "nuclide"
    ],
    "telemetryConsent": "limited"
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-flow, v1.6.0 (inactive)
autocomplete-modules, v1.6.7 (active)
coffee-compile, v0.24.0 (active)
coffee-refactor, v0.7.0 (inactive)
flow-ide, v1.2.4 (inactive)
js-refactor, v0.7.3 (inactive)
language-babel, v2.48.5 (active)
linter, v1.11.18 (active)
linter-eslint, v8.0.0 (active)
linter-flow, v5.5.0 (active)
mac-fix-path, v0.0.0 (active)
nuclide, v0.194.0 (inactive)
refactor, v0.11.2 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.45.0 (inactive)
base16-tomorrow-dark-theme, v1.3.0 (inactive)
base16-tomorrow-light-theme, v1.3.0 (inactive)
one-dark-ui, v1.6.2 (active)
one-light-ui, v1.6.2 (inactive)
one-dark-syntax, v1.5.0 (active)
one-light-syntax, v1.5.0 (inactive)
solarized-dark-syntax, v1.0.5 (inactive)
solarized-light-syntax, v1.0.5 (inactive)
about, v1.7.0 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.13.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.4 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.42.0 (active)
bracket-matcher, v0.82.2 (active)
command-palette, v0.39.0 (active)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.202.2 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.158.8 (active)
metrics, v1.1.2 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.1 (inactive)
settings-view, v0.243.1 (active)
snippets, v1.0.4 (active)
spell-check, v0.68.4 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.2 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.0 (active)
timecop, v0.33.2 (active)
tree-view, v0.210.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.35.0 (active)
wrap-guide, v0.38.2 (active)
language-c, v0.54.0 (active)
language-clojure, v0.22.1 (active)
language-coffee-script, v0.48.0 (active)
language-csharp, v0.13.0 (active)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.43.0 (active)
language-html, v0.47.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (active)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (active)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```
</details>

**Comment by arctelix** [comment]
+1
